src/EditorFeatures/CSharpTest/DocumentationComments/DocumentationCommentTests.cs

```diff
@@ -58,6 +58,31 @@ public void TypingCharacter_Method()
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.DocumentationComments)]
+        public void TypingCharacter_Method_WithVerbatimParams()
```

Comment:
What is the right behavior if the identifier actually requires escaping?  Can you add a test for that case?